docs/source/development/config.rst

```diff
@@ -93,7 +90,7 @@ Profiles
 ========
 
 A profile is a directory containing configuration and runtime files, such as
-logs, connection info for the parallel apps, and your IPython command history.
+logs, connection info, and your IPython command history.
```

Comment:
Yes, it does still contain IPython parallel connection info, and not any other. So while this statement wasn't true for several releases, it is again now.